spec/core/controlplane_spec.rb

```diff
@@ -3,10 +3,19 @@
 require "spec_helper"
 
 describe Controlplane do
-  let!(:fake_config) { Struct.new(:app, :org).new("my-app", "my-org") }
-  let!(:described_instance) { described_class.new(fake_config) }
+  describe "#initialize" do
+    let!(:fake_config) { Struct.new(:app, :org).new("my-app", "my-org") }
```

Comment:
Overall, `let!` with bang is a bad style, especially you "overuse" it. Instead better normal `let` or local var or a method (whatever applicable).

I mean for future, not this PR, try avoiding `let!`

Reply:
Ok, I was using it because for `let!(:app)` because the app needs to be setup before the test begins, so we can't let it be lazily evaluated, but I guess we don't need to use it everywhere.